docs/04_User_Guides/autofix_js_css_resources.md

```diff
@@ -0,0 +1,37 @@
+
+
+# How to apply autoformatting frontend resources
+
+
+# Option1 : Crafty jsLint, tsLint, CssLint options
+`crafty jsLint --fix --preset recommend `
+
+will lint all your files and list the errors
+`npx crafty tsLint <path to files> --preset recommended `
```

Comment:
`tsLint` doesn't exist anymore, only `jsLint` and it's for both JavaScript and TypeScript

Reply:
will remove it :)